oauth/examples/sailfishoauthqmlexample/sailfishoauthqmlexample.qml

```diff
@@ -0,0 +1,116 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.OAuth 1.0
+
+ApplicationWindow {
+    id: window
+    initialPage: Component {
+        Page {
+            id: page
+
+            Rectangle {
+                id: topRect
+                width: parent.width
+                height: parent.height / 2
+                color: "lightsteelblue"
+
+                MouseArea {
+                    anchors.fill: parent
+                    property bool triggered: false
```

Comment:
I see 'WIP' status and I hope it will be used anywhere in the future

Reply:
The triggered property itself?  It just exists to prevent the oauth flow from being started multiple times.

If you mean the example itself: well, yes, it's quite ugly at the moment, exists solely to test that the API works and be quite minimal.  Once we fix the naming etc it would be nice to spend more time making the example more user-friendly also (e.g. show timeline tweets or something, rather than just printing data to console).